packages/create-platformatic/src/index.mjs

```diff
@@ -101,3 +103,10 @@ const createPlatformatic = async (argv) => {
 }
 
 export default createPlatformatic
+
+export {
+  createPackageJson,
+  createGitignore,
+  getVersion,
+  getDependencyVersion
+}
```

Comment:
Can you add a quick test so that we don't remove those by accident?

Reply:
Done!